lib/inferno/dsl/runnable.rb

```diff
@@ -35,6 +35,7 @@ def self.extended(extending_class)
             end
           end
         end
+        @optional = false
```

Comment:
This doesn't seem right.

Reply:
This was to force `optional` to be a boolean,  so that the api would return:

`"optional": false` instead of `"optional": null`.

Fixed by using `!!` in `optional?` in https://github.com/inferno-framework/inferno-core/pull/52/commits/d67e3486edbd323149d9bce3967dcdd795596a6d